package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "debug": "DEBUG=finder,finder:* node index.js"
+    "debug": "DEBUG=finder,finder:* node index.js",
+    "test": "DEBUG=*,-modem,-mocha:* mocha",
+    "test_ci": "DEBUG=*,-modem,-mocha:* mocha ./test/"
```

Comment:
We schould only use an extra script `test_ci` if it does something differently (on the CI platform). _Do you expect that to be the case?

Reply:
Only the path was still different, but i think `mocha` and `mocha ./test` should both start tests from the `test` directory, right?